testing/jupyterlab/test-info-pages/no-tab-trap-initial-page.md

```diff
@@ -0,0 +1,38 @@
+# No Tab Trap on Initial Page
```

Comment:
I wasn't sure what to call these documents. I didn't really like the name "testing scripts" because "scripts" usually means a code file that automates a task, which is the opposite of how we mean it here.

Some alternatives I considered:

- test docs, which I didn't like because I think people might think that they are the docs for the repo or for the whole testing apparatus
- test wiki pages - no because I don't think these are really a wiki, the scope is too narrow
- test descriptions - this is a pretty accurate name but I worry that it's not specific enough because there are other things in the system that somebody might refer to as "test descriptions" for example the text in `test.describe` blocks. I want these things to be called something that when people say the word, they know you're talking about this and not that.
- test summaries - but then I think people will think these are about the results of test output
- test resumes - I kind of liked this one but I also worried if the word resume is too close to summary. Like, again, I don't want folks thinking that this is about summarizing the outputs of the test runs.
- test profiles - but then devs might think we're talking about code performance profiling (CPU, memory usage)

Reply:
I don't have a strong opinion, but I appreciate that you documented possible confusion points. Info and/or info pages works for me.

Reply:
these seem to fall under manual testing scripts (which is the official and industry-wide adopted term)
it is often better to look at what the standard is than creating a new pattern and accounting for all possible confusion from folks

Reply:
Oh interesting! I didn't know that. @trallard could you point me to a variety of bodies of manual testing scripts? That way I can check if the format of those is close enough to the format of what I'm proposing

Reply:
I searched for some good examples with no avail - since most of these tests I have seen are internal documents, I have no real-world examples to share. But generally, they have sections like

- Metadata: author, date updated, version, etc.
- Description: what this is, what does it test, and any compliance/requirements needed
- A table with info like
  - Steps (instructions)
  - Inputs (if any)
  - Expected outcome